modules/elasticsearch/variables.tf

```diff
@@ -516,10 +516,53 @@ variable "kms_key_inaccessible_threshold" {
   default     = "1"
 }
 
+#
+# Redirect-related
+#
+variable "use_redirect" {
+  description = "Indicates whether to use redirect users "
+  default     = false
+}
+
+variable "redirect_route53_zone_id" {
+  description = "Route53 Zone ID to create the Redirect Record in"
+  default     = ""
+}
+
+variable "redirect_domain" {
+  description = "Domain name to redirect"
+  default     = ""
+}
+
+variable "lb_cname" {
+  description = "DNS CNAME for the Load balancer"
+  default     = ""
+}
+
+variable "lb_zone_id" {
+  description = "Zone ID for the Load balancer DNS CNAME"
+  default     = ""
+}
+
+variable "redirect_listener_arn" {
+  description = "LB listener ARN to attach the rule to"
+  default     = ""
+}
```

Comment:
Would it be better to remove the `default` values for these?

Reply:
Good point, edited